src/tcgwars/logic/impl/gen3/PopSeries3.groovy

```diff
@@ -185,6 +202,7 @@ public enum PopSeries3 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 20
+            reduceDamageNextTurn(hp(10), thisMove)
```

Comment:
This only applies to your current active, doesn't it? I think you need to use this method instead
```suggestion
            reduceDamageFromDefendingNextTurn(hp(10), thisMove, defending)
```